USAGE.md

```diff
@@ -64,7 +64,42 @@ layer description, view configuration.
 Saved layer configuration files can be opened in the ATT&CK Navigator to restore a layer you've worked on previously. To
 do so, open a new tab and open the "Open Existing Layer" panel. Then click "Upload from local", and select your saved
 configuration file. Doing so will restore your saved layer to the ATT&CK Navigator. This interface also has a "load from
-URL"input allowing you to open a layer json from a remote source.
+URL" input allowing you to open a layer json from a remote source.
+
+## Upgrading a Layer to the Current Version
+
+A layer uploaded with an outdated version of ATT&CK will prompt you to upgrade to the current version. Choosing "yes" will
+initiate the layer upgrade workflow in the Navigator sidebar. A new, blank layer in the current ATT&CK version and the layer's
+domain will be loaded in a new tab; previous technique annotations can be found in the interface and mapped to techniques in
+the current version, as described below.
```

Comment:
This writing accurately describes what the layer upgrade interface is, but not how it is intended to be used (what the "workflow" actually is). That context is really important to understanding why the different features of the UI are useful. 

> This interface allows users to upgrade an ATT&CK Navigator layer created on a previous version of ATT&CK to the current version of the dataset. The interface allows users to step through each change in the set of techniques in the dataset:
> - The user can see what techniques have been added to the dataset since the layer was created, and annotate them as necessary.
> - The user can see whether the techniques that they had previously annotated have changed, and adjust the annotations accordingly.
> - The user can determine if any annotated techniques have been removed or replaced by new techniques, and in the latter case copy annotations to the replacing technique(s).
> - The user can also verify what techniques haven't been changed since the layer has been created.